drivers/hmis/app/graphql/types/forms/autofill_value.rb

```diff
@@ -19,9 +19,9 @@ class Forms::AutofillValue < Types::BaseObject
     field :sum_questions, [String], 'Link IDs of numeric questions to sum up and set as the value if condition is met', null: true
     field :formula, String, 'Expression with mathematical or logical function defining the value', null: true
 
-    # Condition specifying when to perform this autofill
-    field :autofill_behavior, Types::Forms::Enums::EnableBehavior, null: false
-    field :autofill_when, [Types::Forms::EnableWhen], null: false
+    # Condition specifying when to perform this autofill. If not provided, the autofill will always run.
+    field :autofill_behavior, Types::Forms::Enums::EnableBehavior, null: true
+    field :autofill_when, [Types::Forms::EnableWhen], null: true
```

Comment:
actually this could cause a bug when deployed without the frontend change, hold off

Reply:
https://github.com/greenriver/hmis-frontend/pull/843